brigade/static/css/style.css

```diff
@@ -308,3 +302,56 @@ tbody th {
   }
 
 }
+
+.free-software__logo {
+  margin: 40px 0;
+}
+
+.inkind-card-list {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 50px;
+}
+
+a.inkind-card {
```

Comment:
It would be good to build up the standard ``.card`` component first, and then extend that, so we could have something like ``.card--software`` or ``.card--inkind`` etc. This way we end up with reusable components. 
But this is also something we can address when we clean up and reorganize the style sheet and switch to using SCSS.

Reply:
Agreed. I am leaning towards addressing this when we have multiple cards on the site and have a fledgling style guide going.